src/Stryker.Core/Stryker.Core.UnitTest/DiffProviders/GitDiffProviderTests.cs

```diff
@@ -274,11 +291,15 @@ public void ScanDiffReturnsListofFiles_ExcludingTestFilesInDiffIgnoreFiles()
         public void ScanDiffReturnsListofFiles_ExcludingTestFilesInDiffIgnoreFiles_Single_Asterisk()
         {
             // Arrange
-            var diffIgnoreFiles = new[] { "/c/Users/JohnDoe/Project/*/Test.cs" };
+            var diffIgnoreFiles = new[] { new FilePattern(Glob.Parse("/c/Users/JohnDoe/Project/*/Test.cs"), false, null) };
```

Comment:
Good example of leveraging type safety 👍 

Maybe worth using `FilePattern` in other places, e.g. projects under test and so on?